EDAM_dev.owl

```diff
@@ -37423,7 +37449,41 @@
         <rdfs:comment>NIST Text file format for storing MS&#8725;MS spectra (m&#8725;z and intensity of mass peaks) along with additional annotations for each spectrum. A single MSP file can thus contain single or multiple spectra. This format is frequently used to share spectra libraries.</rdfs:comment>
         <rdfs:label>MSP</rdfs:label>
     </owl:Class>
-	
+    
+
+
+    <!-- http://edamontology.org/format_4041 -->
+
+    <owl:Class rdf:about="http://edamontology.org/format_4041">
+        <rdfs:subClassOf rdf:resource="http://edamontology.org/format_3464"/>"http://www.w3.org/2002/07/owl#DeprecatedClass"/>
```

Comment:
```suggestion
        <rdfs:subClassOf rdf:resource="http://edamontology.org/format_3464"/>
```

Reply:
WTF, it doesn't work to commit this :( (GH askes me in a loop to refresh and try again)

Reply:
Fixed locally & pushed, closing this comment